### ISSUE TITLE
Cannot build cperl 5.28.0

### ISSUE THREAD

**Issue by fperrad** [arsv/perl-cross#83]
There are new variables in `xconfig.h`.
```
xconfig.h:3340:3: error: invalid preprocessing directive #HAS_BUILTIN_CLZ
 # HAS_BUILTIN_CLZ /**/
   ^~~~~~~~~~~~~~~
xconfig.h:3727:3: error: invalid preprocessing directive #HAS_GETCWDNULL
 # HAS_GETCWDNULL  /**/
   ^~~~~~~~~~~~~~
xconfig.h:3734:3: error: invalid preprocessing directive #HAS_GET_CURRENT_DIR_NAME
 # HAS_GET_CURRENT_DIR_NAME  /**/
   ^~~~~~~~~~~~~~~~~~~~~~~~
xconfig.h:3740:3: error: invalid preprocessing directive #HAS_REALPATH
 # HAS_REALPATH  /**/
   ^~~~~~~~~~~~
xconfig.h:5406:3: error: invalid preprocessing directive #USE_NAMED_ANONCV
 # USE_NAMED_ANONCV  /**/
   ^~~~~~~~~~~~~~~~
```

**Comment by arsv** [comment]
Fixed, both 5.28.0 (released Sep 2018) and 5.28.2 (released Mar 2019) should build now.

And I think I'm going to add 5.30.0 which seems to be the latest stable release of cperl.

**Comment by fperrad** [comment]
Good news.
In fact, we miss a lot of __cperl__ releases:
* 5.24.3, 5.24.4
* 5.26.2, 5.26.3, 5.26.4, 5.26.5
* 5.28.0, 5.28.1, 5.28.2
* 5.30.0
